check_trailing_spaces.sh

```diff
@@ -1,14 +1,14 @@
-#!/bin/bash
+#!/bin/sh
 
 files_with_trailing_spaces=$(find . -type f -not -path "./.git/*" -not -path "./dev-tools/vendor/*" -not -path "./vendor/*" -not -path "./tests/Fixtures/*" -exec egrep -nH " $" {} \;)
 
-if [[ $files_with_trailing_spaces ]]
+if [ "$files_with_trailing_spaces" ]
 then
-    echo -e "\e[97;41mTrailing spaces detected:\e[0m"
+    printf '\033[97;41mTrailing spaces detected:\033[0m\n'
     e=$(printf '\033')
-    echo "${files_with_trailing_spaces}" | sed -E "s/^\.\/([^:]+):([0-9]+):(.*[^ ])( +)$/${e}[0;31m - in ${e}[0;33m\1${e}[0;31m at line ${e}[0;33m\2\n   ${e}[0;31m>${e}[0m \3${e}[41m\4${e}[0m/"
+    echo "${files_with_trailing_spaces}" | sed -E "s/^\\.\\/([^:]+):([0-9]+):(.*[^ ])( +)$/${e}[0;31m - in ${e}[0;33m\\1${e}[0;31m at line ${e}[0;33m\\2\\n   ${e}[0;31m>${e}[0m \\3${e}[41m\\4${e}[0m/"
```

Comment:
I copy-pasted this script to upstream:
https://github.com/PHP-CS-Fixer/trailing-whitespace-checker/pull/8/files

with changes in this line, tests on upstream repo would be failing. could you give it a try ?

Reply:
Actually tests pass. There was one weird Travis failure but I restarted it and it passed.

Reply:
nope, take a look, I copied whole file except this one line.

Reply:
Ah sorry, didn't notice that.